sdv/multi_table/base.py

```diff
@@ -359,3 +360,48 @@ def get_learned_distributions(self, table_name):
         """
         synthesizer = self._table_synthesizers[table_name]
         return synthesizer.get_learned_distributions()
+
+    def add_constraints(self, constraints):
+        """Add constraints to the synthesizer.
+
+        Args:
+            constraints (list):
+                List of constraints described as dictionaries in the following format:
+                    * ``constraint_class``: Name of the constraint to apply.
+                    * ``table_name``: Name of the table where to apply the constraint.
+                    * ``constraint_parameters``: A dictionary with the constraint parameters.
+
+        Raises:
+            SynthesizerInputError:
+                Raises when the ``Unique`` constraint is passed.
+        """
+        if self._fitted:
+            warnings.warn(
+                "For these constraints to take effect, please refit the synthesizer using 'fit'."
+            )
+
+        for constraint in constraints:
+            if constraint['constraint_class'] == 'Unique':
+                raise SynthesizerInputError(
+                    "The constraint class 'Unique' is not currently supported."
```

Comment:
I updated the message a little from the issue.

Reply:
maybe add to the end of this sentence "for multi-table synthesizers" so users don't think it's not supported at all